src/clj_jargon/tickets.clj

```diff
@@ -123,6 +149,7 @@
   [^Ticket ticket-obj]
   (> (.getUsesCount ticket-obj) (.getUsesLimit ticket-obj)))
 
+;; unused?
```

Comment:
Should we mark this function as deprecated?

Reply:
I just ran a `git grep` on all our clojure repos, and this `init-ticket-session` function isn't referenced anywhere.
I say it's probably safe to just remove it (de-cruft where possible).

Reply:
Yeah, may as well. I'd actually just marked this to note to myself to check later, but then of course it ended up in the PR and you guys checked instead, heh. Whoops.

I'll pull it out, anyway.